go-controller/pkg/ovn/egressgw.go

```diff
@@ -182,7 +182,8 @@ func (oc *Controller) addPodExternalGW(pod *kapi.Pod) error {
 
 	foundGws, err := getExGwPodIPs(pod)
 	if err != nil {
-		return err
+		klog.Errorf("Error getting exgw IPs for pod: %s, error: %v", pod.Name, err)
```

Comment:
@trozet : should I generate an event here ? We don't seem to do it in other places...

Reply:
right. let me update this PR then.